src/MagnumPlugins/UfbxImporter/UfbxImporter.h

```diff
@@ -247,6 +245,35 @@ can specify a maximum limit using @cb{.ini} maxUvSets @ce,
 @ref Trade-UfbxImporter-configuration "configuration options", note that
 setting any to zero disables loading any tangents etc.
 
+@subsection Trade-UfbxImporter-behavior-animations Animation and skin import
+
+-   FBX represents rotations using Euler angles, which are converted to
+    quaternions on import.
+-   Cubic FBX animation curves are resampled as linear interpolation as time
+    tangents prevent representing them as cubic hermite splines, the resampling
+    rate can be configured via the
+    @cb{.ini} resampleRate @ce @ref Trade-UfbxImporter-configuration "configuration option"
+-   FBX represents rotations as Euler angles so even linear rotations are
+    resampled by default to preserve the original motion, this can be disabled
+    via the
+    @cb{.ini} resampleRotation @ce @ref Trade-UfbxImporter-configuration "configuration option"
+-   Morph targets are not supported
```

Comment:
I assume morph targets are not supported on Magnum yet? I at least couldn't find any way of representing them from UfbxImporter.

Reply:
Yes, not supported, the interfaces are not ready for that yet. It's on my near-term TODO list tho.